server_cmds.go

```diff
@@ -351,19 +365,48 @@ func (s *ServerMethods) PrivilegeKeyAdd(ttype, id1, id2 int, options ...CmdArg)
 
 // OnlineClient represents a client online on a virtual server.
 type OnlineClient struct {
-	ID          int    `ms:"clid"`
-	ChannelID   int    `ms:"cid"`
-	DatabaseID  int    `ms:"client_database_id"`
-	Nickname    string `ms:"client_nickname"`
-	Type        int    `ms:"client_type"`
-	Away        bool   `ms:"client_away"`
-	AwayMessage string `ms:"client_away_message"`
+	// Following variables are always returned by ClientList().
+	ID         int    `ms:"clid"`
+	ChannelID  int    `ms:"cid"`
+	DatabaseID int    `ms:"client_database_id"`
+	Nickname   string `ms:"client_nickname"`
+	Type       int    `ms:"client_type"`
+	// Following variables are optional and can be requested in ClientList() to get extended client information.
+	Away                           *bool   `ms:"client_away"`
+	AwayMessage                    *string `ms:"client_away_message"`
```

Comment:
We should specifically not change these two to maintain compatibility.
```suggestion
	Away                           bool   `ms:"client_away"`
	AwayMessage                    string `ms:"client_away_message"`
```

Reply:
Ok. I stopped using a pointer on these two, but I did make a separate embedded struct for these, since they correspond to the `ClientAway` option. As far as I can understand, this wouldn't change the way how they can be addressed from an OnlineClient struct thus maintaining compatibility. Right?

Reply:
Will need to check this as a duplicate could cause problems.